addon/components/md-btn-dropdown.js

```diff
@@ -25,7 +25,8 @@ export default MaterializeButton.extend({
 
     this.$().dropdown({
       hover: !!this.getWithDefault('hover', false),
-      constrainWidth: !!this.getWithDefault('constrainWidth', true),
+      // jscs:disable requireCamelCaseOrUpperCaseIdentifiers
```

Comment:
make sure you re-enable this rule after the line you want to make an exception for